dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops

```diff
@@ -1257,4 +1257,8 @@ jsp.parse.tld.exception.useraction=Check the JSP Tag Library Descriptor file for
 
 system.java.compiler.not.found=JSPG0304W: The JSP attribute, useJDKCompiler, is set to true, but the Java SDK compiler was not found. The JSP container uses the compiler that is embedded in the product to compile the JSP files in web module: {0}.
 system.java.compiler.not.found.explanation=The JSP container was unable to find the system compiler, which usually indicates that the Java SDK jar file, tools.jar, was not found in the class path.
-system.java.compiler.not.found.useraction=Ensure that the Java SDK jar file, tools.jar, is in the class path and verify that the Java SDK supports the Java Compiler API. See Java Specification Request 199 for more information.
\ No newline at end of file
+system.java.compiler.not.found.useraction=Ensure that the Java SDK jar file, tools.jar, is in the class path and verify that the Java SDK supports the Java Compiler API. See Java Specification Request 199 for more information.
+
+jsp.feature.not.loaded.correctly=CWWJS0001: The JSP [Pages] feature did not load with an acceptable version.
+jsp.feature.not.loaded.correctly.explanation=The JSP [Pages] feature is loaded with a version that corresponds to the Java [Jakarta] EE specification of that JSP [Pages] feature. The feature was loaded with no version defined.
+jsp.feature.not.loaded.correctly.useraction=Check the server.xml file for existence of either a jsp-2.2, jsp-2.3, or pages-3.0 feature. Only one of the listed features should be specified.
```

Comment:
Let me know if any word changes should be made.  I wasn't sure to go with JSP or Pages as this message applies to all features.

Reply:
My only worry here is the useraction as this is going to require continuous updates going forward to add in additional features as they are available. This is something that could be easily missed in the future.

Reply:
I can have it more generic.

Reply:
updated to `jsp.feature.not.loaded.correctly.useraction=Check the server.xml file for existence of either a JSP [Pages] feature. Only one should be specified.`